Guided-registration/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
You can have that license file additionally but the repo is anyways under Apache 2.0 license. So you don't really need that file here, your choice

Reply:
will keep it in